.travis.yml

```diff
@@ -8,15 +8,20 @@ install:
   - go get -u github.com/golang/lint/golint
   - go get -u github.com/Masterminds/glide
   - go get -u golang.org/x/net/context
+  - go get -u github.com/HewlettPackard/gas
   - export GO15VENDOREXPERIMENT=1
   - glide install
 
 script:
+  - gas -skip=management/examples/*.go -skip=*vendor* ./... | tee gas-scan.txt
+  - cat gas-scan.txt
+  - test -z "$((grep 'Severity:\s*HIGH' gas-scan.txt) -eq 0)"
```

Comment:
you don't need `-eq 0` anymore. can you please run these commands locally like `test ...; echo $?` to see what they behave to try out :) I'm getting an email every single time you push a commit.